tests/api2/test_300_nfs.py

```diff
@@ -681,6 +804,10 @@ def test_39_check_nfs_service_protocols_parameter(request):
     assert s['nfsd']["vers3"] == 'y', str(s)
     assert s['nfsd']["vers4"] == 'n', str(s)
 
+    # Confirm setting has taken effect: v4->off, v3->on
+    results = SSH_TEST("rpcinfo -s | grep 100003", user, password, ip)
```

Comment:
Hmmm.  Why 100003?  It's not clear what this is.  Would it be better to search for service name == nfs?
```
root@scale1[~]# rpcinfo -s | awk '$4 == "nfs"'
    100003  4,3       tcp6,tcp                         nfs         superuser
```

Or at least add a define if continuing to use 100003

Same comment applies to duplicate commands below.

Reply:
I agree that 100003 is not descriptive, but it's the well-known unique program number.  There are other services that include 'nfs' in their name, but ' nfs ' should work.  I'll test it.

Reply:
done